style.scss

```diff
@@ -1,2 +1,119 @@
-// @import "@fortawesome/fontawesome-free/css/all.css";
-// @import "node_modules/bootstrap/scss/bootstrap";
+@import "@fortawesome/fontawesome-free/css/all.css";
+@import "node_modules/bootstrap/scss/bootstrap";
+
+
+#primary_nav ul {
+  list-style: none;
+  background: -webkit-linear-gradient(rgb(63, 65, 65), rgb(90, 92, 92));
```

Comment:
We need to see with @kdordevic for the background color of the header, but I don't think it is a gradient.

Reply:
Here we used  rgba(29, 26, 26,0.7) (that is one of our main colors of the site  #1d1a1a with 70% opacity)